index.js

```diff
@@ -37,6 +37,12 @@ class UglifyTask extends TaskKitTask {
         [output]: results.code,
         [`${output}.map`]: results.map
       };
+
+      if (results.error) {
+        console.error(results.error);
```

Comment:
don't need to console error it, if there is an error returned, it will be logged

Reply:
Did that but then it doesn't say the file name

Reply:
what is results.error?  Is it an error object?  Can you find out the structure of it?